docs/sql/ddl.txt

```diff
@@ -826,16 +824,40 @@ To print a user-created table the ``SHOW CREATE TABLE`` statement can be used. T
     ... ) clustered by (first_column) into 5 shards;
     CREATE OK (... sec)
 
-If the ``SHOW CREATE TABLE`` statement gets invoked, the schema of the previous created table gets returned. ::
+The ``SHOW CREATE TABLE`` statement can be used to print the DDL statement of
+already existing user-created doc table in the cluster::
 
     cr> show create table my_table;
-    +-----------------------------------------------+
-    | show create table my_table                    |
-    +-----------------------------------------------+
-    | CREATE TABLE IF NOT EXISTS "my_table" (       |
-    |     "first_column" INTEGER PRIMARY KEY,       |
-    |     "second_column" STRING,                   |
-    |     "third_column" TIMESTAMP                  |
-    | ) CLUSTERED BY ("first_column") INTO 5 SHARDS |
-    +-----------------------------------------------+
-    SELECT 1 row in set (... sec)
\ No newline at end of file
+    +-----------------------------------------------------+
+    | SHOW CREATE TABLE my_table                          |
```

Comment:
this statement still fails, because crash prints incorrectly
will fix that with a crash update